fancyflags/_define_auto_test.py

```diff
@@ -108,5 +108,16 @@ def test_basic_serialization(self):
     flag_values(['./program'] + serialized_args)
     self.assertEqual(flag_values['point'].value(), parsed_point_value)
 
+  def test_disclaimed_module(self):
+    flag_values = flags.FlagValues()
+    _ = _define_auto.DEFINE_auto(
+      'greet', greet, 'help string', flag_values=flag_values)
+    defining_module = flag_values.find_module_defining_flag('greet')
+
+    # the defining module should be the calling module, not the module where
+    # the flag is defined. otherwise the help for a module's flags will not be
+    # printed unless the user uses --helpfull.
```

Comment:
Sorry, could you fix the capitalization here? It's a style guide thing.

Reply:
I was wondering about that, fixed! Also, let me know if you'd like me to squash them – I wasn't sure what the guidelines are on that.

Reply:
There's a public version of our style guide here: https://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar